// built-in types as exceptions
// confuse the parser in throw specifications
foo() throw (int) 
{
}
